### ISSUE TITLE
Error: on NULL from DB

### ISSUE THREAD

**Issue by efureev** [doug-martin/goqu#122]
**Error**: sql: Scan error on column index 1, name "deleted_at": unsupported Scan, storing driver.Value type <nil> into type *time.Time

```go
type User struct {
	Id           string `json:"id" db:"id" goqu:"skipinsert,skipupdate"`
	Email        string    `json:"email" db:"email"`
	PasswordHash []byte    `json:"-" db:"password"`
	Name         string    `json:"name" db:"name"`
	Deleted      *time.Time `json:"-" db:"deleted_at"`
	CreatedAt    time.Time `json:"-" db:"created_at" goqu:"skipinsert,skipupdate"`
}
//...
item := User{}
found, err := database.From(`users`).Where(goqu.C(`email`).Eq(`simple@auth.xx`)).ScanStruct(&item)
if err != nil {
	log.Panicf(`Find: %s`, err)
}
```
//sql: INSERT INTO public.users (id, email, name, password, deleted_at, created_at) VALUES ('4792668f-d192-4dd7-9c47-a70e1e15c602', 'simple@auth.xx', 'Simple', '$2a$08$iNL1iAhLRNSWiJbvuwXTTeTOsBmhZMZnXIlL01twyXwmy9hZeevSi', null, '2019-08-02 11:01:57.908624');

**Comment by doug-martin** [comment]
This is most likely related to the driver you are using, which dialect are you using? 

If you're using `mysql` you probably need to add the parseTime query arg https://stackoverflow.com/questions/45040319/unsupported-scan-storing-driver-value-type-uint8-into-type-time-time

**Comment by efureev** [comment]
postgres. In v6 is working =)

**Comment by doug-martin** [comment]
Ok I found the issue, Im working on a fix.

I should note for both of your use cases I would recommend using the Null types provided by the sql and pq modules. See https://golang.org/pkg/database/sql/#NullString and https://godoc.org/github.com/lib/pq#NullTime

**Comment by doug-martin** [comment]
Fix has been merged and released as `v8.2.2`
